Appraisals

```diff
@@ -6,14 +6,15 @@ RAILS_VERSIONS = %w[
   5.2.5
   6.0.3.6
   6.1.3.1
+  7.0.0
 ]
 
 RAILS_VERSIONS.each do |version|
   appraise "rails_#{version}" do
     gem 'activemodel', version
     gem 'activerecord', version
     
-    if version =~ /^6/
+    if version =~ /^7/ || version =~ /^6/
```

Comment:
Doesn't this kind of matching quickly become cumbersome? wouldn't it be more advantageous to have the last `else` be the newest version?

Reply:
Done :)